pep-0646.rst

```diff
@@ -654,8 +782,8 @@ otherwise imply. Also, we may later wish to support arguments that should not be
 
 We therefore settled on ``TypeVarTuple``.
 
-Behaviour when Type Parameters are not Specified
-------------------------------------------------
+Unspecified Type Parameters: Tuples vs TypeVarTuples
```

Comment:
```suggestion
Unspecified Type Parameters: Tuple vs TypeVarTuple
```
I may sound inconsistent, but the point in this case is that you're writing plurals of specific names, which makes me a bit uncomfortable. (If there was a type Child, would you write Children?)